python/ray/monitor.py

```diff
@@ -288,13 +281,13 @@ def _run(self):
                 self.update_raylet_map()
                 self.autoscaler.update()
 
+            self.get_all_heartbeat()
             # Process a round of messages.
             self.process_messages()
 
-            # Wait for a heartbeat interval before processing the next round of
-            # messages.
-            time.sleep(
-                ray._config.raylet_heartbeat_timeout_milliseconds() * 1e-3)
+            # Wait for a autoscaler update interval before processing the next
+            # round of messages.
+            time.sleep(AUTOSCALER_UPDATE_INTERVAL_S)
```

Comment:
Is the monitor.py doing anything else useful that may be affected by adding this sleep? Or is it only handling autoscaler metrics?

Reply:
Seems monitor.py is for autoscaler, nothing else.